### ISSUE TITLE
Please allow unused variables / parameters starting with an '_'

### ISSUE THREAD

**Issue by mcm1957** [ioBroker/ioBroker.eslint-config#20]
Old eslint allowed to mark parameters and exception paramaters to be named _xxx to avoid unused paramaters error.
Please reenable this featiure as otherwise either many repos must use modified local rules or code changes will be required.

Feature should be able with the following modification:

```
    {
        rules: {
            "@typescript-eslint/no-unused-vars": [
            "error", {
                "argsIgnorePattern": "^_",
                "varsIgnorePattern": "^_",
                "caughtErrorsIgnorePattern": "^_"
                }
            ]
        },
    },
```

Please check and add to standard rules.

NOTE: eventually unused variables starting with an underscore are not really needed, but at least unused paramaters are often present to show the complete set of parameters passed to the routine.

**Comment by mcm1957** [comment]
Hmm - das diable steht in den ts rules schon drinnen:
https://github.com/ioBroker/ioBroker.eslint-config/blob/9bdcbd31d044a7cd55dae5223bca09ea63f64d76/eslint.config.mjs#L64

Das scheint nur für *.js fiels nicht zu ziehen.
Ev. ziehen da auch andere Dinge nicht für js ?

**Comment by foxriver76** [comment]
fixed with v1.0.0

**Comment by mcm1957** [comment]
Its working as expected - conformed at nsclient adapter.
